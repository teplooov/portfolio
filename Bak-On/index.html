<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<link rel="stylesheet" href="css/bootstrap.min.css">
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/animate.css">
	<link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">


	<title>Document</title>
</head>
<body>
	<header class="header d-flex" id="header">
		<div class="container">
			<div class="row ">
				<div class="col-lg-3">
					<div class="logo">
						<span class="bold ">YOU-PS.Ru</span>
						<p class="logo__text">Best PSD files</p>
					</div>
				</div>
				<div class="col-lg-9 ">
					<nav>
						<ul class="menu d-flex justify-content-around " >
							<li class="menu__item "><a href="#gome">home</a></li>
							<li class="menu__item"><a href="#portfolio">portfolio</a></li>
							<li class="menu__item"><a href="#about">about</a></li>
							<li class="menu__item"><a href="#contact">contact</a></li>
						</ul>	
					</nav>	
				</div>
			</div>
		</div>
	</header>
	<div class="main">
		<div class="offer">
		<div class="container">
			<div class="row">
					<div class="col-lg-12 d-flex flex-column">
						<h1 class="offer__title">WE Build <span class="bold">Brand</span></h1>
						<p class="text__offer mr-auto ml-auto">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.</p>
						<div class="btn">
						<a class="btn__link" href="#">LEARN MORE</a>
					</div>
					<div class="down ">
						<a href=""><i class="fas fa-chevron-circle-down"></i></a>
					</div>
					</div>
				</div>
			</div>
		</div>
		<div class="portfolio" id="portfolio">
			<div class="title__portfolio ml-auto mr-auto">
				<div class="container">
					<div class="row d-flex ">
						<div class="col-lg-12 ">
						<h2>Porfolio</h2>
						<p class="text__portfolio">
							Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						</p>
					</div>
				</div>
					<div class="row ">
						<div class="col-lg-12">
							<nav>
								<ul class="portfolio__menu d-flex justify-content-center " >
									<li class="portfolio__item"><a href="">all</a></li>
									<li class="portfolio__item"><a href="">web design</a></li>
									<li class="portfolio__item"><a href="">graphic design</a></li>
									<li class="portfolio__item"><a href="">flat design</a></li>
								</ul>	
							</nav>		
						</div>
					</div>
					<div class="portfolio__item">
						<div class="row d-flex justify-content-around">
							<div class="col-lg-3"><img src="img/img-1.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-2.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-3.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-4.png" alt=""></div>
						</div>
					</div>
					<div class="portfolio__item">
						<div class="row d-flex justify-content-around">
							<div class="col-lg-3"><img src="img/img-5.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-6.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-1.png" alt=""></div>
							<div class="col-lg-3"><img src="img/img-7.png" alt=""></div>
					</div>
				</div>
				</div>
				</div>
			</div>
			<div class="about__us" id="about">
				<div class="container">
					<div class="row">
						<div class="col-lg-12">
							<h2>About Us</h2>
							<p class="text__portfolio">
							Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						</p>
						</div>
					</div>
					<div class="row d-flex justify-content-center mr-auto ml-auto">
						<div class="col-lg-3">
							
							<div class="title__ava wow FadeInLeftBig">
								<img src="img/ava-1.png" alt="">
								 <h3>Melannie Doe</h3>
								 <span><i>Project Manager</i></span>
								<p>Ut wisi enim ad minim veniam,quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.
								</p>
							</div>
						</div>
						<div class="col-lg-3">
							<div class="title__ava wow FadeInLeftBig ">
								<img src="img/ava-3.png" alt="">
								<h3>Jhonnie Doe</h3>
								<span><i>Graphic Designer</i></span>
								<p>Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.
								</p>
							</div>
						</div>
						<div class="col-lg-3">
							<div class="title__ava wow FadeInLeftBig">
								<img src="img/ava-2.png" alt="">
								<h3>Daniel Doe</h3>
								<span><i>Web Designer</i></span>
								<p>Ut wisi enim ad minim veniam,quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.</p>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="contact-us" id="contact">
				<div class="container">
					<div class="row d-flex ">
						<div class="col-lg-12 ">
						<h2>Contact Us</h2>
						<p class="text__contact">
							Lorem ipsum dolor sit amet, consectetur adipisicing elit.
						</p>
					</div>
				</div>
					<div class="form__contact">
					<div class="row d-flex ">
						<div class="col-lg-6">
							<form action="get" class="">
								<div class="form-group d-flex">
									<input type="text" class="form__info" placeholder="Your Name">
									<input type="mail" class="form__info" placeholder="Your Email">
								</div>
								<div class="form-group">
   									 <textarea class="form-control" id="exampleFormControlTextarea1" rows="8"p placeholder="Your message" ></textarea>
  								</div>
  								<div class="submit">
  									<input type="submit" value="SUBMIT" class="btn__link btn__submit">
  								</div>
							</form>
						</div>
							<div class="col-lg-6">	
								<div class="contact__info">
								<h4>INFORMATION</h4>
								<p class="adres">
								<i class="fas fa-map-marker-alt"></i>Bak-One Company <br>	
								12345 Hollywood Bvld Street <br>
								Los Angeles, California</p>
								<p class="contact"><i class="fas fa-phone-square"></i> +444 (Phone) 123456 <br><br>
								<i class="fas fa-fax"></i> +123 (FAX) 0011223 <br><br>
								<i class="far fa-envelope"></i> info@bak-onecompany.com <br>
								</p>
							</div>
						</div>
						</div>						
						</div>
					</div>
				</div>
				<footer class="footer">
					<div class="container">
					<div class="row">
						
					</div>
				</div>
				</footer>
			</div>
			</div>
			</div>			
		</div>
	</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>	
<script src="js/js.js"></script>
<script src="js/wow.min.js"></script>
<script>
  new WOW().init();
</script>
<script src="js/bootstrap.min.js"></script>
	
</body>
</html>