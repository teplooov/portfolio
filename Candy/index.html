<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

	<link rel="stylesheet" href="style/bootstrap.min.css">
	<link rel="stylesheet" href="style/style.css">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<link href="https://fonts.googleapis.com/css?family=Fresca&display=swap" rel="stylesheet">

	<title>Document</title>
</head>
<body>
	<header class="header">
		<div class="container">
			<div class="wrap">
				<nav class="navbar navbar-expand-lg navbar-light ">
  					<a class="navbar-brand" href="#">Navbar</a>
  						<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
   							 <span class="navbar-toggler-icon"></span>
 						 </button>
  					<div class="collapse navbar-collapse" id="navbarNav">
   					 	<ul class="navbar-nav">
     					 	<li class="nav-item ">
        						<a class="nav-link" href="#">Главная <span class="sr-only">(current)</span></a>
      						</li>
      						<li class="nav-item">
        						<a class="nav-link" href="#">О нас</a>
      						</li>
      						<li class="nav-item">
        						<a class="nav-link" href="#">Услуги</a>
      						</li>
      						<li class="nav-item">
        						<a class="nav-link" href="#">Контакты</a>
      						</li>
   					 	</ul>
  					</div>
				</nav>
			</div>
			<div class="wrap d-flex">
				<div class="col-lg-12">
					<div class="container">
						<div class="preload d-flex">
							<h1>Вата-кат</h1>
							<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Temporibus voluptates recusandae cumque, ex officia ea.</p>
						</div>
					</div>
				</div>
			</div>
			
		</div>
	</header>
	<main>
		<section class="info">
			<div class="container">
				<h2>О нас</h2>
				<div class="info__item">
					<img src="img/step.jpg" alt="" class="img1">
					<p class="info__text">Lorem ipsum dolor sit amet, consectetur adipisicing e	lit. Eos nostrum rem placeat numquam. Qui quod aliquid aliquam repellendus atque soluta explicabo ab quibusdam sunt, a, quisquam debitis sint incidunt 	omnis similique et quas eos. Voluptatem ullam id nulla sed facere ad itaque hic aliquam ratione soluta incidunt, doloremque! Totam vero, quod non quam temporibus magni reprehenderit officiis esse voluptatibus facere ab error odio amet quaerat incidunt dicta repellendus natus aliquid voluptas blanditiis, fugit ipsam nobis obcaecati nostrum. Pariatur aspernatur eius eos fugiat, ad vitae, debitis cum natus ipsam, saepe ea! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias quisquam inventore consequuntur! Delectus officia, repellendus cumque optio perferendis, ad, nesciunt incidunt reprehenderit eum distinctio totam! Ipsa numquam iste exercitationem in libero unde doloribus quidem sapiente facilis quos perspiciatis, velit tempore! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Similique, amet! Minima quos ad ratione culpa rerum accusamus iure earum architecto tempora expedita amet minus ipsam eum reiciendis, voluptatem sint quam illum consectetur perspiciatis quibusdam, itaque alias est modi. Rerum, ipsa ex quo sequi enim totam mollitia pariatur quis sed. Enim!</p>
				</div>
			</div>
		</section>

		<section class="services">
			<div class="container">
				<h2>Наши Услуги</h2>
				<div class="services__list">
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
				</div>
				<div class="services__list">
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
				</div>

			</div>
		</section>
		<section class="gallerey">
			<div class="container-fluid">
				<div id="carouselExampleInterval" class="carousel slide" data-ride="carousel">
  					<div class="carousel-inner">
   					 <div class="carousel-item active" data-interval="10000">
      					<img src="img/step.jpg" class="d-block w-100" alt="...">
   					 </div>
   				    <div class="carousel-item" data-interval="2000">
      					<img src="img/step.jpg" class="d-block w-100" alt="...">
   					</div>
   					 <div class="carousel-item">
      					<img src="img/step.jpg" class="d-block w-100" alt="...">
   						</div>
  					</div>
  					<a class="carousel-control-prev" href="#carouselExampleInterval" role="button" data-slide="prev">
   						 <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    					<span class="sr-only">Previous</span>
  					</a>
  					<a class="carousel-control-next" href="#carouselExampleInterval" role="button" data-slide="next">
   					 <span class="carousel-control-next-icon" aria-hidden="true"></span>
    				<span class="sr-only">Next</span>
  					</a>
				</div>
			</div>
		</section>
		<section class="contact">
			<div class="container">
				<form action="get" class="form__group">
					
				</form>
			</div>
		</section>
	</main>
	<script src="js/jquery-3.4.1.min.js"></script>
	<script src="js/bootstrap.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>