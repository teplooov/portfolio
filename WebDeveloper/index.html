<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<link rel="stylesheet" href="css/normalize.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="css/style.css">
	<title>Document</title>
</head>
<body>
	<header class="header">
		<div class="container">
		<nav><div class="header__logo"><p>Здесь будет логотип</p></div>
			<div class="top__nav" id="mytopnav">
				<a href="#" id="menu" class="icon">&#9776</a>
				<a class="nav__link" href="#">Главная</a>
				<a class="nav__link" href="#">О нас</a>
				<a class="nav__link" href="#">Услуги</a>
				<a class="nav__link" href="#">Отзывы</a>
				<a class="nav__link" href="#">Контакты</a>
			</div>
		</nav>
		<div class="main__welcome">
			<h1>ВАТА<span class="mycompany">-КАТ</span></h1>
			<p>We are a digital agency with years of experience and with extraordinary people</p>
		</div>
</div>
	</header>
	<main>

		<section class="info">
			<div class="container">
				<h2>О нас</h2>
				<div class="info__item">
					<img src="img/step.jpg" alt="" class="img1">

					<p class="info__text">Lorem ipsum dolor sit amet, consectetur adipisicing e	lit. Eos nostrum rem placeat numquam. Qui quod aliquid aliquam repellendus atque soluta explicabo ab quibusdam sunt, a, quisquam debitis sint incidunt 	omnis similique et quas eos. Voluptatem ullam id nulla sed facere ad itaque hic aliquam ratione soluta incidunt, doloremque! Totam vero, quod non quam temporibus magni reprehenderit officiis esse voluptatibus facere ab error odio amet quaerat incidunt dicta repellendus natus aliquid voluptas blanditiis, fugit ipsam nobis obcaecati nostrum. Pariatur aspernatur eius eos fugiat, ad vitae, debitis cum natus ipsam, saepe ea! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias quisquam inventore consequuntur! Delectus officia, repellendus cumque optio perferendis, ad, nesciunt incidunt reprehenderit eum distinctio totam! Ipsa numquam iste exercitationem in libero unde doloribus quidem sapiente facilis quos perspiciatis, velit tempore! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Similique, amet! Minima quos ad ratione culpa rerum accusamus iure earum architecto tempora expedita amet minus ipsam eum reiciendis, voluptatem sint quam illum consectetur perspiciatis quibusdam, itaque alias est modi. Rerum, ipsa ex quo sequi enim totam mollitia pariatur quis sed. Enim!</p>
				</div>
			</div>
		</section>
		<section class="services">
			<div class="container">
				<h2>Наши Услуги</h2>
				<div class="services__list">
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
				</div>
				<div class="services__list">
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
					<div class="services__item">
						<img src="img/—Pngtree—sweets candy festive candy cartoon_3854978.png" alt=""><p class="services__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus amet delectus unde distinctio inventore minus ab et labore animi blanditiis.</p>
					</div>
				</div>

			</div>
		</section>
	</main>
	<script src="js/script.js"></script>
</body>
</html>