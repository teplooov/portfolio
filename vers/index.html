<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="stylesheet" href="owl/assets/owl.carousel.min.css">
<link rel="stylesheet" href="owl/assets/owl.theme.default.min.css">
	<link rel="stylesheet" href="style/style.css">
	<link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
	<title>webdesign</title>
</head>
<body>
	<header>
		<div class="header-container">
		<div class="logo">
			<img src="img/LOGO.png" alt="logo">
		</div>
		<input type="checkbox" id="menu-checkbox">
		<nav role="navigation">
			<label for="menu-checkbox" class="toggle-button" data-open="MENU" data-close="CLOSE" onclick></label>
			<ul class="main-menu" id="main-menu">
				<li><a href="#">home</a></li>
				<li><a href="#">portfolio</a></li>
				<li><a href="#">blog</a></li>
				<li><a href="#">about us</a></li>
				<li><a href="#">contact</a></li>
			</ul>
			
		</nav>
	</div>
	</header>
	<main>
		<section class="top-slid">
			<div class="top-container">
					<div class=""><h2>WE BELIVE</h2></div>
				<div class=""><h2>IN QUALITY DESIGN</h2></div>
			<div class="">
				<p>Any creative project is unique</p>
			</div>
						<div class="">
				<p>and should be provided with </p>
			</div>
						<div class="">
				<p>the appropriate quality</p>
			</div>
		</div>	
		</section>
		<section class="specialize">
			<div class="specialize-container">
				<h3>WE CREATE QUALITY DESIGNS.</h3>
				<p>We specialize in Web Design / Development and Graphic Design</p>
				<div class="specialize-cards">
					<div class="card">
						<div class="top-card">
							<img src="img/card1.png" alt="">
						</div>
						<div class="bottom-card">
							<p class="first-card">Redesigning<br> With Personality</p>
						</div>
					</div>
					<div class="card">
						<div class="top-card">
							<img src="img/WEB DEVELOPMENT.png" alt="">
						</div>
						<div class="bottom-card">
							<p class="two-card">Lorem ipsum dolor sit amet,<br>consectetur.</p>
						</div>
					</div>
					<div class="card">
						<div class="top-card">
							<img src="img/WEB DESIGN.png" alt="">
						</div>
						<div class="bottom-card">
							<p class="two-card">Lorem ipsum dolor sit amet,<br> consectetur.</p>
						</div>
					</div>
					<div class="card">
						<div class="top-card">
							<img src="img/GRAPHIC DESIGN.png" alt="">
						</div>
						<div class="bottom-card">
							<p class="two-card">Lorem ipsum dolor sit amet,<br> consectetur.</p>
						</div>
					</div>
			</div>
		</div>
		</section>
		<section class="portfolio">
			<h4>OUR WORK</h4>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laboriosam reprehenderit corporis, veniam voluptatibus sunt.</p>
			<div class="toggles">
				<button id="showall">SHOW ALL</button>
				<button id="webdew">WEB DEVELOPMENT</button>
				<button id="webdesign">WEB DESIGN</button>
				<button id="graphd">GRAPHIC DESIGN</button>
			</div>
			<div class="posts">
				<div class="post webdew"><img src="img/Portfolio work 1.png" alt=""></div>
				<div class="post webdesign"><img src="img/Portfolio work 10.png" alt=""></div>
				<div class="post webdesign"><img src="img/Portfolio work 11.png" alt=""></div>
				<div class="post graphd"><img src="img/Portfolio work 2.png" alt=""></div>
				<div class="post webdesign"><img src="img/Portfolio work 4.png" alt=""></div>
				<div class="post graphd"><img src="img/Portfolio work 5.png" alt=""></div>
				<div class="post graphd"><img src="img/Portfolio work 6.png" alt=""></div>
				<div class="post webdew"><img src="img/Portfolio work 6.png" alt=""></div>
				<div class="post graphd"><img src="img/Portfolio work 6.png" alt=""></div>
				<div class="post webdew"><img src="img/Portfolio work 6.png" alt=""></div>
						
			</div>
		</section>
		<section class="brand-slider">
			<h4>OUR CLIENTS</h4>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Provident error magnam ducimus iste recusandae.</p>
			<div class="owl-carousel">
				<div class=""><img src="img/Client 3.jpg" alt=""></div>
				<div class=""><img src="img/Client 4.jpg" alt=""></div>
				<div class=""><img src="img/Client 5.jpg" alt=""></div>
				<div class=""><img src="img/Client 6.jpg" alt=""></div>
				<div class=""><img src="img/Client 7.jpg" alt=""></div>
				<div class=""><img src="img/Client 8.jpg" alt=""></div>
				<div class=""><img src="img/Client 3.jpg" alt=""></div>
				<div class=""><img src="img/Client 5.jpg" alt=""></div>
			</div>
		</section>
	</main>
	<footer>
		<div class="footer-contaner">
		<div class="about-us">
			<h4>About us</h4>
			<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, 
				sed diam nonummy nibh euismod tincidunt ut 
				laoreet dolore magna aliquam erat volutpat. Ut wisi 
				enim ad minim veniam, quis nostrud exerci tation ullamcorper 
				suscipit lobortis nisl ut aliquip ex ea commodo consequat.
			</p>
			<a href="">Learn more</a>
			<h4>Photo Stream</h4>
			<div class="images">
				<img src="img/img 1.jpg" alt="">
				<img src="img/img 2.jpg" alt="">
				<img src="img/img 3.jpg" alt="">
				<img src="img/img 4.jpg" alt="">
				<img src="img/img 5.jpg" alt="">
				<img src="img/img 6.jpg" alt="">
			</div>	
		</div>

		<div class="tweets">
			<h4>Latest tweets</h4>	
			<p class="twwts_info"><img src="img/Twitter birs icon.png" alt=""> Check Out Dtbaker'<span class="yellow">@Arduino</span> Sales Notification <span class="yellow">#System </span></p>
			<p class="twwts_info"><img src="img/Twitter birs icon.png" alt=""> Community Superstar Winner – <span class="yellow">#OrganicBeeMedia</span></p>
			<p class="twwts_info"><img src="img/Twitter birs icon.png" alt=""> Clean <span class="yellow">#Websites</span> Designs for Your <span class="yellow">#Inspiration</span></p>
			<p class="twwts_info"><img src="img/Twitter birs icon.png" alt=""> Cute Online <span class="yellow">#Shops</span></p>
			<p class="twwts_info"><img src="img/Twitter birs icon.png" alt=""> Download 40 vector <span class="yellow">#icons</span> package for <span class="yellow">#FREE</span></p>
			<div class="Social-Connecting">
				<h4>Social Connecting</h4>
				<div class="social-img">
					<img src="img/dribbble.png" alt="">
					<img src="img/facebook.png" alt="">
					<img src="img/google plus.png" alt="">
					<img src="img/twitter.png" alt="">
					<img src="img/google plus.png" alt="">
					<img src="img/google plus.png" alt="">
					<img src="img/facebook.png" alt="">
					<img src="img/twitter.png" alt="">

				</div>
			</div>
		</div>

		<div class="contacts">
			<h4>Contact info</h4>
			<p class="contacts-info"><img src="img/Adress.png" alt="">Address: 12569 Saint Patrick des Prés, 85000 Paris, France</p>
			<p class="contacts-info"><img src="img/Phone.png" alt="">Phone: +38 045 845-45-78<br>+38 045 845-45-79</p>
			<p class="contacts-info"><img src="img/email.png" alt="">E-mail:<a href="#"><span class="yellow">freeforwebdesign@gmail.com</a></p></span>
			<div class="follow">
				<h4>Follow Us</h4>
				<p class="">Lorem Ipsum is simply dummy text of the printing <br>
and typesetting industry.</p>
		</div>
		</div>
	</div>

	</footer>

	<script src="js/jquery-3.4.1.min.js"></script>
	<script src="owl/owl.carousel.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>