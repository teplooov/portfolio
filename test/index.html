<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="css/bootstrap.min.css">
	<link rel="stylesheet" href="css/style.css">
	<title>Document</title>
</head>
<body>
<header class="header" id="header">
	<div class="container">
		<div class="row">
			<div class="col-lg-6">
		<div class="logo d-flex">
			<img src="img/logo_header.svg" alt="">
			<p class="logo_text">Lorem ipsum dolor sit amet. <br>
			Lorem ipsum dolor sit. </p>
		</div>
		</div>
		<div class="col-lg-6">
		<nav class="menu">
			<div class="menu__list">
				<a href="" class="menu__item">Установка</a>
				<a href="" class="menu__item">Возможности</a>
				<a href="" class="menu__item">Стоимость</a>
				<a href="" class="menu__item">Регистрация</a>
			</div>
		</nav>
	</div>
		</div>
		<div class="row">
			<div class="col-lg-6">
				<div class="promo">
					<h1>Принимайте обращение клиентов из разных каналов в одном сервисе</h1>
					<a href="" class="btn">Начать обрабатывать заявки с Webim</a>
				</div>
			</div>
			<div class="col-lg-6">
				<div class="promo__img">
					<img src="img/img_01_x2.png" alt="">
				</div>
			</div>
		</div>
	</div>
</header>
<script src="js/jquery-3.4.1.min.js"></script>
<script src="js/bootstrap.min.js"></script>
<script src="js/main.js"></script>
</body>
</html>