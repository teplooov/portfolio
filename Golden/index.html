<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <title>layout</title>
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/style.css">
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
</head>
<body>
  <header id="header">
      <div class="header__one">
      <div class="logo"><img src="img/logo.png" alt=""></div>
      <div class="menu">
        <ul class="menu__list">
          <li class="menu__item"><a class="go_to" href="#header">home</a></li>
          <li class="menu__item"><a class="go_to" href="#services">services</a></li>
          <li class="menu__item"><a class="go_to" href="#portfolio">portfolio</a></li>
          <li class="menu__item"><a class="go_to" href="#about">about</a></li>
          <li class="menu__item"><a class="go_to" href="#social">social</a></li>
        </ul>
        </div>
      </div>
      <div class="title">
        <div class="title__first">Welcome to our studio!</div>
        <h1>It’s nice to Meet you</h1>
      </div>
      <div class="button"><a href="">tell me more</a></div>
  </header>
    <main>
      <section Id="services">
        <div class="services__title">
          <h2>SERVICES</h2>
          <p>Proin iaculis purus consequat sem cure. </p>
        </div>
        <div class="services__list">
          <div class="services__item">
            <img src="img/icon1.png" alt="">
            <h3>E-Commerce</h3>
            <p>Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
          <div class="services__item">
            <img src="img/icon2.png" alt="">
            <h3>Responsive Web</h3>
            <p>Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
          <div class="services__item">
            <img src="img/icon3.png" alt="">
            <h3>Web Security</h3>
            <p>Proin iaculis purus consequat sem cure
              digni ssim. Donec porttitora entum suscipit
              aenean rhoncus posuere odio in tincidunt.</p>
          </div>
        </div>
      </section>
      <section id="portfolio">
        <div class="container">
          <div class="portfolio__title">
            <h2>OUR PORTFOLIO</h2>
            <p>Proin iaculis purus consequat sem cure. </p>
          </div>
          <div class="portfolio__list">
            <img src="img/img1.jpg" alt="">
            <img src="img/img2.jpg" alt="">
            <img src="img/img3.jpg" alt="">
          </div>
        </div>
      </section>
      <section id="about">
        <div class="about__title">
          <h2>About Us</h2>
          <p>Proin iaculis purus consequat sem cure. </p>
        </div>
        <div class="about__list">
          <div class="about__item1">
            <div class="item__info">
             <h3>July 2010 Our Humble Beginnings</h3>
              <p>Proin iaculis purus consequat sem cure
                  digni ssim. Donec porttitora entum suscipit
                  aenean rhoncus posuere odio in tincidunt. Proin
                  iaculis purus consequat sem cure digni
                  ssim. Donec porttitora entum suscipit.</p>
            </div>
            <div class="item__img">
              <img src="img/about1.jpg" alt="">
            </div>
          </div>
                   <div class="about__item2">
            <div class="item__img">
              <img src="img/about2.jpg" alt="">
            </div>
            <div class="item__info2">
              <h3>January 2011 Facing Startup Battles</h3>
              <p>Proin iaculis purus consequat sem cure
                  digni ssim. Donec porttitora entum suscipit
                  aenean rhoncus posuere odio in tincidunt. Proin
                  iaculis purus consequat sem cure digni
                  ssim. Donec porttitora entum suscipit aenean
                  rhoncus posuere odio in tincidunt.
              </p>
            </div>
          </div>
                   <div class="about__item1">
            <div class="item__info">
              <h3>December 2012 Enter The Dark Days</h3>
              <p>Proin iaculis purus consequat sem cure
                  digni ssim. Donec porttitora entum suscipit
                  aenean rhoncus posuere odio in tincidunt. Proin
                  iaculis purus consequat sem cure digni.
                </p>
            </div>
            <div class="item__img">
              <img src="img/about3.jpg" alt="">
            </div>
          </div>
                   <div class="about__item2">
            <div class="item__img">
              <img src="img/about4.jpg" alt="">
            </div>
            <div class="item__info2">
              <h3>FEBRUARY 2014 Our Triumph</h3>
              <p>Proin iaculis purus consequat sem cure
                  digni ssim. Donec porttitora entum suscipit
                  aenean rhoncus posuere odio in tincidunt. Proin
                  iaculis purus consequat sem cure digni
                  ssim. Donec porttitora entum suscipit aenean.
                </p>
            </div>
          </div>
        </div>
      </section>
      </main>
      <footer id="social">
        <div class="social__item">
          <a href=""><i class="fab fa-vk"></i></a>
          <a href=""><i class="fab fa-twitter"></i></a>
          <a href=""><i class="fab fa-instagram"></i></a>
        </div>
        <p>© Copyright 2014 FreebiesXpress.com</p>
      </footer>
  <script src="js/jquery-3.3.1.min.js"></script>
  <script src="js/js.js"></script>
</body>
</html>